ci-operator/step-registry/cucushift/downgrade/hypershift/worker-node/toimage/cucushift-downgrade-hypershift-worker-node-toimage-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: cucushift-downgrade-hypershift-worker-node-toimage
+  from: ansible
```

Comment:
I guess we do not need to use `ansible` image here, because I did not see the step calls any ansible commands.

Reply:
okay, done, change to `upi-installer`

Reply:
How about using verification-tests image to keep consistent with cucushift-upgrade-toimage step:
```
  from_image:
    namespace: ci
    name: verification-tests
    tag: latest
```

Reply:
BTW, can you also update `cucushift-upgrade-hypershift-worker-node-toimage` together.

Reply:
https://github.com/openshift/verification-tests/blob/master/tools/openshift-ci/Dockerfile#L67
For the verification-tests image, it installs Ansible. I can confirm that we don't need to use the Ansible part.

Reply:
For hypershift upgrade and downgrade step, we primarily utilized OC commands, so using the UPI installer should cover it.

Reply:
Or even to be cleaner, you can use `from: cli` for the hypershift upgrade/downgrade step.

Reply:
Thanks a lot, done

Reply:
Do you like to run a new pr-rehearse job to confirm the update of image still works.